addons/mail/static/src/core/common/quick_reaction_menu.js

```diff
@@ -27,13 +26,17 @@ export class QuickReactionMenu extends Component {
     setup() {
         this.toggle = useRef("toggle");
         this.store = useState(useService("mail.store"));
-        this.picker = usePopover(EmojiPicker, {
-            position: "bottom-end",
-            popoverClass: "o-mail-QuickReactionMenu-pickerPopover shadow-sm",
-            animation: false,
-            onPositioned: (el, { direction, variant }) =>
-                el.classList.add(`o-popover--${direction[0]}${variant[0]}`),
-        });
+        this.picker = useEmojiPicker(
+            null,
```

Comment:
API starts great when the first required param gets null/undefined half of the time. 🤔

Reply:
Agreed, but I'm not touching useEmojiPicker, just making the quick reaction menu use it, like everywhere else in the code. Same for the comment below, I agree that passing an element would have been better, especially since the dropdown API does it this way.